Docs/Configuration.tex

```diff
@@ -3288,12 +3287,12 @@ \subsection{Security Properties}\label{miscsecurityprops}
   \href{https://support.apple.com/en-us/HT208330}{\texttt{Full Security}} of Apple
   Secure Boot.
 
-  To start using personalised Apple Secure Boot one will have to reinstall the
-  operating system or personalise it. Until the operating system is personalised
-  one will only be able to load macOS DMG recovery. If DMG recovery is missing,
+  To start using personalised Apple Secure Boot, the  operating system will have
+  to be reinstalled or personalised. Unless the operating system is personalised,
```

Comment:
It sounds a bit wordy here. How about just *Unless personalised*?

Reply:
Repeating "OS" provides clarity and don't think it is particularly wordier than what applies across the rest of the document.

Reply:
I would try to avoid repetitions where available. How about *Until the completion of personalisation,*?